PORTS.md

```diff
@@ -0,0 +1,37 @@
+# Ports Documentation
+
+Eclipse Thingweb uses a cloud instance provided by Eclipse Foundation to host its services on [plugfest.thingweb.io](http://plugfest.thingweb.io/). 
+In order to function as a TCP/UDP server, we have some ports open to the public Internet.
+These are documented below, alongside the service using the port.
+
+## TCP Ports
+
+| TCP PORT | Service |
+|----------|---------|
+|   80     |  Apache Web Server  |
+|   8080   |  Available       |
+|   8081   |  Available       |
+|   8082   |  Available        |
+|   8083   |  Example Things       |
+|   8084   |  Available       |
+|   8085   |  Available       |
+|   8086   |  Available       |
+|   8087   |  Available       |
+|   8088   |  WoT FX UI       |
+|   8089   |  Available       |
+|   5685   |  Available       |
+
+
+## UDP Ports
+
+| UDP PORT | Service |
+|----------|---------|
+|   5680   |  Available       |
+|   5681   |  Available       |
+|   5682   |  Available       |
+|   5683   |  Example Things       |
```

Comment:
Maybe we can (should?) add that it is CoAP ?

Reply:
Done :)